src/coreclr/jit/emitxarch.cpp

```diff
@@ -11733,10 +11841,10 @@ BYTE* emitter::emitOutputRR(BYTE* dst, instrDesc* id)
                         regMaskTP regMask;
                         regMask = genRegMask(reg1) | genRegMask(reg2);
 
-                        // r1/r2 could have been a GCREF as GCREF + int=BYREF
-                        //                            or BYREF+/-int=BYREF
-                        assert(((regMask & emitThisGCrefRegs) && (ins == INS_add)) ||
-                               ((regMask & emitThisByrefRegs) && (ins == INS_add || ins == INS_sub)));
+// r1/r2 could have been a GCREF as GCREF + int=BYREF
+//                            or BYREF+/-int=BYREF
+// assert(((regMask & emitThisGCrefRegs) && (ins == INS_add)) ||
+//        ((regMask & emitThisByrefRegs) && (ins == INS_add || ins == INS_sub)));
```

Comment:
You plan on re-enabling this assert, right?

Reply:
Yes, it shouldn't have been included in the PR at all 😄

Reply:
Fixed.